catalystwan/session.py

```diff
@@ -399,7 +432,12 @@ def request(self, method, url, *args, **kwargs) -> ManagerResponse:
         if self.request_timeout is not None:  # do not modify user provided kwargs unless property is set
             _kwargs.update(timeout=self.request_timeout)
         try:
-            response = super(ManagerSession, self).request(method, full_url, *args, **_kwargs)
+            if self._limiter is None:
+                response = super(ManagerSession, self).request(method, full_url, *args, **_kwargs)
+            else:
+                delayed_request = lambda: super(ManagerSession, self).request(method, full_url, *args, **_kwargs)
+                response = self._limiter.send_request(delayed_request)
```

Comment:
You can create dummy limiter and do it in this way: 

```suggestion
            with self._limiter:
                response = super(ManagerSession, self).request(method, full_url, *args, **_kwargs)
```